meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source /usr/sbin/resin-vars
+
+SUPERVISOR_IMAGE_ID=$(docker inspect $SUPERVISOR_IMAGE | jq '.[0].Id')
+SUPERVISOR_CONTAINER_IMAGE_ID=$(docker inspect resin_supervisor | jq '.[0].Image')
+
+if [ "$SUPERVISOR_IMAGE_ID" -eq "$SUPERVISOR_CONTAINER_IMAGE_ID" ]; then
+	docker start resin_supervisor
```

Comment:
I'd like a fallback to removing the container if starting it fails - it can help with auto-recovery in some cases (I think that's why we initially switched to always removing)